docs/dev-environment-setup.md

```diff
@@ -0,0 +1,256 @@
+# Setting up the Development Environment
+
+## 1. Repository and Version Control
+
+- Ensure you have push access (most likely by being added to the **@bcgov/cas-developers** GitHub team) to [bcgov/cas-ciip-portal](https://github.com/bcgov/cas-ciip-portal).
```

Comment:
Also needs to be added to the bcgov/cas team, which is required for the CircleCI security context (needed to run some jobs which need access to our openshift cluster)